src/applications/accreditation/21a/pages/01-personal-information-chapter/personalInformation.js

```diff
@@ -0,0 +1,30 @@
+import { cloneDeep } from 'lodash';
+import {
+  dateOfBirthSchema,
+  dateOfBirthUI,
+  fullNameSchema,
+  fullNameUI,
+  titleUI,
+} from 'platform/forms-system/src/js/web-component-patterns';
+
+const fullNameMiddleInitialUI = cloneDeep(fullNameUI());
```

Comment:
Nit, instead of cloneDeep, could you use shallow copy (more optimized):
```
const fullNameMiddleInitialUI = { ...fullNameUI() };
fullNameMiddleInitialUI.middle['ui:title'] = 'Middle initial';
```

Reply:
I like that!